<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bit Shift Operatpr</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DeleteFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SavePath">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Path</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>H:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaskValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0x00000001</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="XOR">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Logical</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="AND">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>AND</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Logical</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="OR">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>OR</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">Logical</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Modificate">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SaveRule">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Индексировать ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="Reverse">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Result_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Reverse_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Logical"/>
 </buttongroups>
</ui>
